desc/configuration.py

```diff
@@ -1049,10 +1052,7 @@ def p_l(self, p_l):
     @property
     def iota(self):
         """Rotational transform (iota) profile."""
-        try:
-            return self._iota
-        except AttributeError:
-            return None
+        return self._iota
```

Comment:
This change broke several tests throwing an `AttributeError`. The Equilibrium now always has these attributes assigned, but either iota or current will be `None` and the HDF5 IO is not properly handling `None` types. I would prefer to fix that root cause of the issue before adding this try-except back in.

Reply:
We still need the try-except blocks because we check whether a profile exists by querying the attribute (`if self.current is None`). I'd actually prefer to remove the try-except blocks and use `if hasattr(self, "current")` instead of the `None` check.